### PR TITLE
[Am-23] Create admin login page

### PR DESCRIPTION
**Pull request by sr-meh**
**SOLVES: #13** 

## **Changes:**

> -  add a new page to sign up admin (with styles)

**desktop view**

image

**mobile view**

image

### CODE REVIEW

**Review by luisarmandodv — DISMISSED** [review]
LGTM :shipit:

File: app/assets/stylesheets/layout/_new.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.log-in__section {
+  padding-top: 150px;
+  display: flex;
+  justify-content: center;
```
Comment (jaimelr) [inline app/assets/stylesheets/layout/_new.scss]: Add properties alphabetically.

File: app/assets/stylesheets/layout/_new.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+.new_user {
+  padding: 50px 0;
+
+  input {
+    border: 1px solid #ccc;
```
Comment (jaimelr) [inline app/assets/stylesheets/layout/_new.scss]: If `#ccc` is a color from the theme you should add it as a variable in `_colors.scss`. What do you think?

File: app/controllers/admin_controller.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+# To blkalbala
```
Comment (jaimelr) [inline app/controllers/admin_controller.rb]: Is this comment necessary?

File: app/views/devise/sessions/new.html.erb [hunk 4]
```diff
@@ -1,26 +1,28 @@
...
+    <% end -%>
 
-<%= render "devise/shared/links" %>
+    <div class="actions">
+      <%= f.submit "Log in" %>
+    </div>
+  <% end %>
+  <%= render "devise/shared/links" %>
+  </div>
+</div>
```
Comment (jaimelr) [inline app/views/devise/sessions/new.html.erb]: This file should be HAML, shouldn't it?

**Review by jaimelr — DISMISSED** [review]
Just a couple of comments.

**Review by jaimelr — DISMISSED** [review]
Nice! LGTM!

**Review by jaimelr — APPROVED** [review]

File: app/views/devise/sessions/new.html.haml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+.log-in__section.container
+  .log-in__content
+    %h2 Log in
+    = form_for(resource, as: resource_name, url: session_path(resource_name)) do |f|
+      .field
+        = f.label :email
+        %br/
```
Comment (8geonirt) [inline app/views/devise/sessions/new.html.haml:7]: Don't use `br`s to separate elements, use CSS instead

**Pull request merged**